src/main/java/net/sf/jabref/exporter/OOCalcDatabase.java

```diff
@@ -67,7 +66,7 @@ public OOCalcDatabase(BibtexDatabase bibtex, Set<String> keySet) {
             }
         }
 
-        entries = new SortedList(entryList, new FieldComparatorStack(comparators));
+        entries = new SortedList<BibtexEntry>(entryList, new FieldComparatorStack<BibtexEntry>(comparators));
```

Comment:
No, a correct solution would look like this:

`entries = new SortedList<>(entryList, new FieldComparatorStack<>(comparators));`

You still need the empty `<>`. But you do not have to retype the generic type, just indicate that the generic type of the variable declaration must be used.